model/labels/labels_dedupelabels.go

```diff
@@ -417,6 +417,13 @@ func (ls Labels) WithoutEmpty() Labels {
 	return ls
 }
 
+// ByteSize returns the approximate size of the labels in bytes.
+// String header size is ignored because it should be amortized to zero.
+// SymbolTable size is also not taken into account.
```

Comment:
Not taking SymbolTable size is unfortunate, since it could reveal some mistakes / opportunities where the table could be re-used.  However you'd have to redesign this function as taking a whole set of labels in order to get that right.
And it is likely low-value as we didn't get `dedupelabels` sufficiently efficient.

Reply:
I am adding string header size into calculation in 469ae3c18229e98c96e7378614757081c5d79ec9, to make it consistent with suggestion on slicelabels.

Reply:
Agreed with both your comments here, however you are responding to something I didn't say.
My comment is about SymbolTable.